crates/bevy_ecs/src/event.rs

```diff
@@ -103,20 +107,21 @@ enum State {
 ///
 /// # Details
 ///
-/// [Events] is implemented using a double buffer. Each call to [Events::update] swaps buffers and
-/// clears out the oldest buffer. [EventReader]s that read at least once per update will never drop
-/// events. [EventReader]s that read once within two updates might still receive some events.
-/// [EventReader]s that read after two updates are guaranteed to drop all events that occurred
+/// [`Events`] is implemented using a variation of a double buffer strategy.
+/// Each call to [`update`](Events::update) swaps buffers and clears out the oldest one.
+/// [`EventReader`]s will read events from both buffers.
+/// [`EventReader`]s that read at least once per update will never drop events.
+/// [`EventReader`]s that read once within two updates might still receive some events.
+/// [`EventReader`]s that read after two updates are guaranteed to drop all events that occurred
```

Comment:
I'd make this a list but it's not a big thing

Reply:
Yeah, fair, forgot that existed